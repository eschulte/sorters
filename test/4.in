94 38 42 30 19 90 42
