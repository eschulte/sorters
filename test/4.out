19 30 38 42 42 90 94
